wii7800/src/wii/wii_atari_menu.cpp

```diff
@@ -711,14 +716,19 @@ static void wii_read_game_list( TREENODE *menu )
  */
 static void wii_read_save_state_list( TREENODE *menu )
 {
-  DIR_ITER *ssdir = diropen( WII_SAVES_DIR );
+  DIR *ssdir = opendir( WII_SAVES_DIR );
   if( ssdir != NULL)
-  {   
+  { 
+    struct dirent *dent;
     struct stat statbuf;
     char ext[WII_MAX_PATH];
     char filepath[WII_MAX_PATH];
-    while( dirnext( ssdir, filepath, &statbuf ) == 0 ) 
-    {            
+    while ((dent = readdir(ssdir)) != NULL)
+    {
+      char* filepath = dent->d_name;
+      char path[WII_MAX_PATH];
+      sprintf(path,"%s/%s", WII_ROMS_DIR, filepath);

```

Comment:
This should be WII_SAVES_DIR

Reply:
@raz0red in the end you merged with the `WII_ROMS_DIR` instead of `WII_SAVES_DIR`?
I can try the binary this WE.

Reply:
I went ahead and merged your request, then made the fix. It should be correct now. I also made quite a number of other changes.

Reply:
I went ahead and merged it, and made the fix as a separate commit. I restructured things a bit, and am in the process of resolving the audio issues (popping, etc.).

Reply:
great, thanks!